Dockerfile

```diff
@@ -8,24 +8,11 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-# This is a Dockerfile allowing to build dashboard by using a docker container.
-# Build step: $ docker build -t eclipse-che-dashboard .
-# It builds an archive file that can be used by doing later
-#  $ docker run --rm eclipse-che-dashboard | tar -C target/ -zxf -
-FROM node:8.16.0
+FROM docker.io/httpd:2.4.43-alpine AS registry
```

Comment:
I would not remove the current Dockerfile or rename the current one

because it's two different flows: building with docker and serving the target folder

BTW we could do both build and serve on the same Dockerfile

Reply:
That was my first iteration. Later I switch to native build because I want to cache npm modules. What do you prefer to create new Dockerfile or adopt existed? I 70/30 to create new? WDYT?

Reply:
new and merge/drop once all ppl are happy